src/main/java/svc/data/citations/datasources/tyler/transformers/MunicipalityIdTransformer.java

```diff
@@ -0,0 +1,49 @@
+package svc.data.citations.datasources.tyler.transformers;
+
+import java.sql.ResultSet;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import svc.data.jdbc.BaseJdbcDao;
+import svc.logging.LogSystem;
+import svc.models.Municipality;
+import svc.types.HashableEntity;
+
+@Component
+public class MunicipalityIdTransformer extends BaseJdbcDao {
+
+	public HashableEntity<Municipality> lookupMunicipalityId(String datasource, String datasourceMunicipalityIdentifier) {
```

Comment:
For `datasource`, still pass the enum. Do not .toString() it until you need to at the DB level. We should try to avoid passing around the string representation since it makes it more fragile.